doc/nrf/glossary.rst

```diff
@@ -0,0 +1,447 @@
+
+Global Navigation Satellite System (GNSS)
+   A satellite navigation system with global coverage.
+   The system provides signals from space transmitting positioning and timing data to GNSS receivers, which use this data to determine location.
+
+Global Positioning System (GPS)
+   A satellite-based radio navigation system that provides its users with accurate location and time information over the globe.
+
+Hardware Flow Control (HWFC)
+   A handshaking mechanism used to prevent a buffer overflow in the receiver (in embedded computing use cases).
+   When the transmission baud rate is high enough for data to appear faster than it can be processed by the receiver, the communicating devices can synchronize with each other, using RTS and CTS pins.
+
+Host Controller Interface (HCI)
+   Standardized communication between the host stack (e.g., a PC or mobile phone OS) and the controller (the Bluetooth IC).
+   This standard allows the host stack or controller IC to be swapped with minimal adaptation.
+
+Human Interface Device (HID)
+   Type of a computer device that interacts directly with, and most often takes input from, humans and may deliver output to humans.
+   The term "HID" most commonly refers to the USB-HID specification.
+   This standard allows the host stack or controller IC to be swapped with minimal adaptation.
+
+Integrated Circuit (IC)
+   A semiconductor chip consisting of fabricated transistors, resistors, and capacitors.
+
+Integrated Development Environment (IDE)
+   A software application that provides facilities for software development.
+
+Internet Control Message Protocol (ICMP)
+   The control protocol of the IP stack that enables the establishment of reachability, routes, and so on.
+   This protocol is an integral part of any IP but is unique as it is not a transport protocol to exchange data between hosts.
+
+Internet Protocol version 4 (IPv4)
+   The fourth version in the development of the Internet Protocol (IP).
+   It is the communications protocol that provides an identification and location system for computers on networks.
+   It routes most traffic on the Internet.
+
+Internet Protocol version 6 (IPv6)
+   The latest version of the Internet Protocol (IP).
+   It is the communications protocol that provides an identification and location system for computers on networks and routes traffic across the Internet.
+
+IPv4 address
+   A numerical label that is used to identify a network interface of a computer or other network node participating in an IPv4 computer network.
+
+IPv6 address
+   An alphanumerical label that is used to identify a network interface of a computer or other network node participating in an IPv6 computer network.
+
+Kconfig file
+   A configuration file for a module, written in the Kconfig language syntax.
```

Comment:
You can also have Kconfig files for samples.

Reply:
fixed